### PR TITLE
FizzBuzz Kata Solution Implemented

### PR DESCRIPTION
**Pull request by marcos-x86**

### CODE REVIEW

File: src/test/java/org/fundacionjala/coding/marcoslara/FizzBuzzTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    /**
+     * Basic test for the number fifteen.
+     */
+    @Test
+    public void fizzBuzzOfFifteen() {
+        assertEquals(FizzBuzz.printNumber(15), "FizzBuzz");
+    }
+
+    /**
+     * FOR THE STAGE TWO
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/marcoslara/FizzBuzzTest.java]: You need to add more unit tests for stage two

File: src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    /**
+     * This method find a digit in a number.
+     *
+     * @param n     is the number to be evaluated.
+     * @param digit is the number to find.
+     * @return the find result.
+     */
+    private static boolean isContaining(int n, int digit) {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java]: please rename this method name. I don't understand :P

Comment (marcos-x86) [reply src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java]: Renamed (y).

File: src/test/java/org/fundacionjala/coding/marcoslara/FizzBuzzTest.java (hunk 1 above)
Comment (marcos-x86) [reply src/test/java/org/fundacionjala/coding/marcoslara/FizzBuzzTest.java]: More Unit Test added (y).

File: src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    /**
+     * This method return the representation of numbers multiples of three and five.
+     *
+     * @param n is the number to be evaluated.
+     * @return the string representation based in the multiples.
+     */
+    private static String printWord(int n) {
+        StringBuilder stringBuilder = new StringBuilder();
+        if (n % 3 == 0 || hasThisNumber(n, THREE)) {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java]: Can we use the constant here ?

File: src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+    /**
+     * This method return the representation of a number based on the rules
+     * described.
+     *
+     * @param n is the number to be evaluated.
+     * @return the number representation.
+     */
+    public static String printNumber(int n) {
+        return n % 3 == 0 || n % 5 == 0 || hasThisNumber(n, THREE) || hasThisNumber(n, FIVE)
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java]: Can we use the constants here ?

File: src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java (hunk 3 above)
Comment (marcos-x86) [reply src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java]: Yes, we can (y).

File: src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java (hunk 4 above)
Comment (marcos-x86) [reply src/main/java/org/fundacionjala/coding/marcoslara/FizzBuzz.java]: Yes (y).

**Comment by carledriss** [comment]
LGTM!

**Pull request merged**